extensions/amp-ad/0.1/test/test-amp-ad-xorigin-iframe-handler.js

```diff
@@ -50,32 +56,26 @@ describe('amp-ad-xorigin-iframe-handler', () => {
   });
 
   describe('init() returned promise', () => {
-    let iframe;
     let initPromise;
 
     describe('if render-start is implemented', () => {
 
       let noContentSpy;
 
-      beforeEach(() => {
+      beforeEach(done => {
```

Comment:
It's a nodeback (an async callback that takes an error for the first parameter, if one occurred).